src/recap.c

```diff
@@ -146,20 +113,20 @@ void log_cmdline_error(const char *format, ...) {
   va_list argList;
 
   fflush(stdout); /* Flush any pending stdout */
+  fflush(stderr);
 
-  fprintf(stdout, "Command-line usage error: ");
+  fprintf(stderr, "Command-line usage error: ");
   va_start(argList, format);
-  vfprintf(stdout, format, argList);
+  vfprintf(stderr, format, argList);
   va_end(argList);
 
   fflush(stderr); /* In case stderr is not line-buffered */
   exit(EXIT_FAILURE);
 }
 
-void process_app_options(
-    int argc, char *argv[], uint8_t *verbosity, char **pcap_path,
-    char **out_path, char **ifname, int *pipe,
-    enum SQLITE_TRANSACTION_TYPE *sqlite_transaction_type) {
+void process_app_options(int argc, char *argv[], uint8_t *verbosity,
+                         char **pcap_path, char **out_path, char **ifname,
+                         int *pipe, int *capture, int *transaction) {
```

Comment:
Is it worth using C99 boolean types instead?

See https://en.cppreference.com/w/c/types/boolean

Reply:
I can change to bool. But I'm an old timer :) Should have used char instead.

Reply:
Done.